### ISSUE TITLE
Don't always count an HTTP error 404 as a change

### ISSUE THREAD

**Issue by schildbach** [thp/urlwatch#286]
I want to monitor a not yet existing URL for something to appear. Unfortunately currently each request will trigger a change, regardless of if it just again returned the same 404 error. So in short, if the error doesn't change, then please don't count it as a change. If it goes from error to no error (or the other way round), then it would be a change indeed.

**Comment by schildbach** [comment]
btw. I tried the Debian package 2.14-1.

**Comment by thp** [comment]
You can set the config option `display` / `error` to `false`, although this at the moment is a global setting, so any other errors will also be ignored and not shown.

**Comment by schildbach** [comment]
@cfbao It's not a full workaround. Yes it seems to detect going from 404 to some content. But not the other way round.

**Comment by cfbao** [comment]
Would an option like `treat_error_as_content` work then? So errors wouldn't be handled in a special way. When you have consecutive errors, no notice would be fired as long as they have the same messages. But changing between error & non-error content would trigger a change report.

**Comment by schildbach** [comment]
Yes, that's exactly what I tried to propose. At least for an HTTP 404 error.

(I'm not so sure about other HTTP errors; I think for example the 5xx category should not trigger a content change.)

**Comment by schildbach** [comment]
HTTP 403 (Forbidden) is also one of these errors that should count as content. For example, for one of my watched pages the content changed from HTTP 404 to HTTP 403. That's a very important information I'd like to be notified about.
